src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
The use of `static alive = [];` is not supported in all JavaScript environments. Consider using a constructor function or a static method to initialize this property to ensure compatibility.